q2cli/tools.py

```diff
@@ -19,26 +19,35 @@ def tools():
 
 
 @tools.command(name='export',
-               short_help='Export data from a QIIME 2 Artifact or '
-                          'Visualization.',
-               help="Export data from a QIIME 2 Artifact or Visualization. "
-                    "Exporting extracts the data stored in an Artifact or "
-                    "Visualization and will support exporting to multiple "
-                    "formats in the future. For now, the data is exported in "
-                    "the format it is stored within the Artifact or "
-                    "Visualization. Use 'qiime tools extract' to extract the "
-                    "Artifact or Visualization's entire archive.")
-@click.argument('path', type=click.Path(exists=True, file_okay=True,
-                                        dir_okay=False, readable=True))
-@q2cli.option('--output-dir', required=True,
-              type=click.Path(exists=False, file_okay=False, dir_okay=True,
+               short_help='Export data from a QIIME 2 Artifact',
+               help="Exporting extracts the data stored in an Artifact"
```

Comment:
Try:
`"Exporting extracts (and optionally transforms) data stored inside an Artifact or Visualization. Note that Visualizations cannot be transformed with --output-format."` reflow as needed for PEP8.

Reply:
@ebolyen, thanks for reviewing the PR, I will go ahead and make these changes!